### PR TITLE
potential new config file options

### PR DESCRIPTION
**Pull request by tim-thacker-nullify**
## Description

I'd like to get anyone's input on these or any others we should add

potential new config file options for:
- allowlisting CVEs
- configuring scheduled digest messages

## Checklist

- [ ] I have added one of the `patch`, `minor`, `major` or `no-release` labels
- [ ] I have linked an issue from this repository using the **Development** option
- [ ] I have performed a self-review of my own code
- [ ] I have checked for redundant or commented out code
- [ ] I have commented my code where I can't make it self-documenting
- [ ] I have made corresponding changes to the documentation
- [ ] I have added any appropriate tests

### CODE REVIEW

File: data/nullify.yaml [hunk 1]
```diff
@@ -2,3 +2,20 @@ severity_threshold: medium
...
+    - new-findings
+    - new-cves
+    - new-secrets-exposed
+    targets:
+    - type: email
+      emails: [monitoring@nullify.ai]
+    - type: slack
+      channel_id: "123456"
+dependencies:
+  ignore:
```
Comment (yushindkang) [inline data/nullify.yaml:18]: once it's merged in a customer's repo, will it go and create comments above sast vulns in the code? And if customers whitelist using inline comments, will the config file be updated automatically too?

Comment (tim-thacker-nullify) [reply data/nullify.yaml:18]: this isnt for SAST, its for SCA/CVEs.
for SAST we do code comments. cant leave comments in most package files e.g. package.json

Comment (yushindkang) [reply data/nullify.yaml:18]: ah I was under the impression it's for both sast and cve. Cool

Comment (tim-thacker-nullify) [reply data/nullify.yaml:18]: check out the PR for SAST in the-force

**Review by yushindkang — APPROVED** [review]

**Review by RahulTripathi0401 — APPROVED** [review]

**Pull request merged**